app/partials/send.jade

```diff
@@ -128,11 +127,6 @@
                          when="{'1': '{{block | translate}}', 'other': '{{blocks | translate}}' }")
             span ).
           span.has-error(ng-show="blockIdx > 6" translate="SMALL_FEE_WARNING")
-      //- QR Scanner 
-      .form-group.row(ng-hide="!cameraIsOn")
-        label.col-sm-2.col-xs-12(translate="QR")
-        .col-sm-10.col-xs-12
-          bc-qr-reader(active="cameraRequested", on-result="processURLfromQR" on-error="onError" camera-status="cameraIsOn")
```

Comment:
removing processURLfromQR call also removes call to which removes the call to applyPaymentRequest, so if you scan a QR with an amount attached the amount field won't update.